scripts/run-parity-and-deploy.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+cd /
+./start.sh &
+
+cd /maker-docker-poa
+
+sleep 10 &
+
+npm install
+npx tsc
+./fetch-sai-contracts.sh
+node output/deployment/compileContracts.js
+node output/tools/generateContractInterfaces.js
+
+# We need to give parity some time to come up. The above commands usually take plenty of time, but this waits on a
+# Backgrounded sleep that was kicked off earlier, just in case the above steps finish too quickly
+wait %2
```

Comment:
Consider instead putting something like this in `deployContracts.js`
```typescript
	while (true) {
		try {
			console.log(`attempting to connect to Ethereum node at ${jsonRpcAddress}...`)
			await provider.getBlockNumber()
			break
		} catch (error) {
			await sleep(1000)
			continue
		}
	}
```

That is how I deal with this problem over in our contract deployment script.